csv_files/csv_util.py

```diff
@@ -0,0 +1,12 @@
+import time
+from constants.definitions import CSV_PATH
+
+def write_state_to_csv(predicted_state):
+    with open(CSV_PATH + '/datastore.csv', 'a') as fd:
+        fd.write(
+            str(predicted_state[0])[1:-1] + ',' + str(predicted_state[1])[1:-1] + ',' + str(predicted_state[2])[1:-1] + '\n')
+    time.sleep(0.001)
```

Comment:
why is there a sleep here and not in write_phase_to_csv?

Reply:
The time.sleep is to make the state/location on the map for the GUI load in at a normal speed.

Looking at it again since the sleep is used to slow down move to target node calculations for the GUI, it will also sleep the phases that are written so its consistent. Also tho moved this back into robot since this sleep should only apply to move to target node